activejob/lib/active_job/execution_state.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module ActiveJob
+  module ExecutionState # :nodoc:
+    extend ActiveSupport::Concern
+
+    included do
+      def perform_now
+        I18n.with_locale(locale) do
+          Time.use_zone(timezone) { super }
+        end
+      end
+    end
```

Comment:
```suggestion
    def perform_now
      I18n.with_locale(locale) do
        Time.use_zone(timezone) { super }
      end
    end
```
No need for `included` here.